CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # CHANGELOG
 
+<<<<<<< HEAD
 ### v2.03
  - Split LPA `my*` location codes into `lp*` and `pa*` codes, representing Circulating and Research locations.
+=======
+### romcom-2.0-prelease 
+ - remove deliverableTo from M2 items
+ - add deliverableToResolution to all locations.csv entries
+ - update serializer to accomodate new property
+>>>>>>> SCC-3608/romcom-2.0
```

Comment:
I think we should tag this one v2.1 since all of this work can be enabled immediately (after testing). When you create the other PR to make M1 locations `.requestable`, we should tag that "romcom-2.0-prelease" since we want to hold on merging that until launch